### PR TITLE
feat: boat-test-tube-baby

### LINKED ISSUE #959: feat: Boat test tube baby
Let's make this pattern:
image

Please work at this file: https://github.com/TroyTae/game-of-life/blob/master/src/life/oscillator/boat-test-tube-baby.json

If you wanna tackle this issue, please leave comment.  
I'll assign to you!  

Are you first time for contributing this repo?  
Please read this [guide](https://github.com/TroyTae/game-of-life/blob/master/.github/CONTRIBUTING.md) :)

### PR DESCRIPTION
**Pull request by Mrskillful**
boat-test-tube-baby dot image
resolve #959

### CODE REVIEW

**Comment by Mrskillful** [comment]
Just realized I left the object name as "life" and not "boat test tube baby". 
I'm gonna edit this now.

File: src/life/oscillator/boat-test-tube-baby.json [hunk 1]
```diff
@@ -1,8 +1,13 @@
 {
   "title": "Boat test tube baby",
-  "life": [
-    [0, 0, 0],
-    [0, 1, 0],
-    [0, 0, 0]
+  "Boat test tube baby": [
```
Comment (TroyTae) [inline src/life/oscillator/boat-test-tube-baby.json:3]: You don't need to change property key name
Please recover this line to `life`

**Comment by Mrskillful** [comment]
What in the world... 
I'm going to bed and redoing it tommorow.

File: src/life/oscillator/boat-test-tube-baby.json (hunk 1 above)
Comment (Mrskillful) [reply src/life/oscillator/boat-test-tube-baby.json:3]: Ah okay.
Making the changes now.

Comment (Mrskillful) [reply src/life/oscillator/boat-test-tube-baby.json]: Ah okay,
Let me make the changes.

**Comment by Mrskillful** [comment]
I changed it to a draft in case I couldn't get to it. 
Though I fixed my mistakes, should be fine now.

If anything, let me know.

**Comment by Mrskillful** [comment]
Thanks for letting me contribute.

Now to show my friends my dot image of a uterus.

**Pull request merged**